designs/l1_l4_security/l1_l4_security.md

```diff
@@ -0,0 +1,75 @@
+# L1-L4 security leveraging OpenStack security groups
+This document describes the notifications available for OpenStack security groups
+
+## Assignees
+Intel's OSC core team
+
+## Background
+OSC will support single page view to manage OpenStack security groups. User will be able to manage the OpenStack security groups and security group rules from OSC.
+
+## Constraints and Assumptions
+No addtional security groups are configured by user or admin directly on OpenStack.
+
+## Notifications provided by OpenStack
```

Comment:
What is the purpose of these notifications and user permissions for the implementation of this feature? Can you provide more details? The document seems to lack something that connects this info with the overall feature. Also, can you delete 'XYZ Integration Point' and place this session at the location in the document? The _XYZ Integration Point_ is not a real header, just a placeholder for a session that describes the  communication/integration of OSC with other external endpoints and I think and fits for this.